packages/react-scripts/template/README.md

```diff
@@ -430,8 +429,8 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 ```diff
    "scripts": {
-     "build-css": "node-sass src/ -o src/",
-     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
+     "build-css": "node-sass-chokidar src/ -o src/",
+     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
```

Comment:
@michaelwayman : Thanks. That's exactly I want. 
I tried the second way, edit web-pack-config, run eject to add scss loader and eventually it failed.
The node-sass-chokidar's way works fine, so I think something likes scss loader will make node-sass-chokidar better :)

P/S: I'm using VS, and config it to hide css files.